extensions/libarchive/MergeContent.h

```diff
@@ -32,17 +32,21 @@ namespace nifi {
 namespace minifi {
 namespace processors {
 
-#define MERGE_STRATEGY_BIN_PACK "Bin-Packing Algorithm"
-#define MERGE_STRATEGY_DEFRAGMENT "Defragment"
-#define MERGE_FORMAT_TAR_VALUE "TAR"
-#define MERGE_FORMAT_ZIP_VALUE "ZIP"
-#define MERGE_FORMAT_FLOWFILE_STREAM_V3_VALUE "FlowFile Stream, v3"
-#define MERGE_FORMAT_FLOWFILE_STREAM_V2_VALUE  "FlowFile Stream, v2"
-#define MERGE_FORMAT_FLOWFILE_TAR_V1_VALUE  "FlowFile Tar, v1"
-#define MERGE_FORMAT_CONCAT_VALUE "Binary Concatenation"
-#define MERGE_FORMAT_AVRO_VALUE "Avro"
-#define DELIMITER_STRATEGY_FILENAME "Filename"
-#define DELIMITER_STRATEGY_TEXT "Text"
+namespace MergeContentOptions {
+constexpr const char *MERGE_STRATEGY_BIN_PACK = "Bin-Packing Algorithm";
+constexpr const char *MERGE_STRATEGY_DEFRAGMENT = "Defragment";
+constexpr const char *MERGE_FORMAT_TAR_VALUE = "TAR";
+constexpr const char *MERGE_FORMAT_ZIP_VALUE = "ZIP";
+constexpr const char *MERGE_FORMAT_FLOWFILE_STREAM_V3_VALUE = "FlowFile Stream, v3";
+constexpr const char *MERGE_FORMAT_FLOWFILE_STREAM_V2_VALUE = "FlowFile Stream, v2";
+constexpr const char *MERGE_FORMAT_FLOWFILE_TAR_V1_VALUE = "FlowFile Tar, v1";
+constexpr const char *MERGE_FORMAT_CONCAT_VALUE = "Binary Concatenation";
+constexpr const char *MERGE_FORMAT_AVRO_VALUE = "Avro";
+constexpr const char *DELIMITER_STRATEGY_FILENAME = "Filename";
+constexpr const char *DELIMITER_STRATEGY_TEXT = "Text";
+constexpr const char *ATTRIBUTE_STRATEGY_KEEP_COMMON = "Keep Only Common Attributes";
+constexpr const char *ATTRIBUTE_STRATEGY_KEEP_ALL_UNIQUE = "Keep All Unique Attributes";
+}
```

Comment:
we could add a comment at the closing bracket about what namespace is being closed, maybe some empty lines before and after the body of the namespace

Reply:
https://google.github.io/styleguide/cppguide.html#Namespaces
https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace